### ISSUE TITLE
Add concept of Administrator

### ISSUE THREAD

**Issue by rossfuhrman** [rubyforgood/CharterForCompassion#65]
Several of our other issues reference the idea of Amanda the website Administrator, but we do not have any way of specifying that a given user is an admin.
I'm thinking maybe we can just add an 'admin' flag on the user record so we can determine which users are an admin.

**Comment by jacobherrington** [comment]
@rossfuhrman Would it make sense to use a gem like devise here?

**Comment by jacobherrington** [comment]
That would knock out #66 as well.

**Comment by mobjohnson** [comment]
Was looking into this, and I agree with CanCanCan and maybe Rolify too. Ross just implemented 'One Admin to Rule Them All' for orgs, but there will be more than one admin for both Charter and the separate Orgs, and these two get us to there. Not sure that is MVP though. Railscasts include some videos about implementing the original CanCan that I found helpful.

**Comment by seanmarcia** [comment]
I'd recommend Pundit :)

**Comment by rossfuhrman** [comment]
I do think using a gem for authorization is probably a good idea.
I would also be in favor of Pundit (over CanCanCan or Rolify), and that's mainly because I have a bit of experience with Pundit, but none with the others.
As for having multiple admins for a given Organization, I think that will be needed at some point (and like you said @mobjohnson, maybe not for the MVP), but that is a separate issue.

**Comment by mobjohnson** [comment]
Pundit looks good!

**Comment by jacobherrington** [comment]
@mobjohnson are you working on this? I'm happy to if it's up for grabs.

**Comment by mobjohnson** [comment]
@jacobherrington no, i am not. go for it!!! :) and thanks for your help. it will be AWESOME.

**Comment by cattywampus** [comment]
It looks like this issue was addressed by PR #76, should this be closed or is there still more work that needs to be done?
